locust/rpc/zmqrpc.py

```diff
@@ -17,9 +17,13 @@ class Server(BaseSocket):
     def __init__(self, host, port):
         context = zmq.Context()
         self.receiver = context.socket(zmq.PULL)
+        self.receiver.setsockopt(zmq.TCP_KEEPALIVE, 1)
+        self.receiver.setsockopt(zmq.TCP_KEEPALIVE_IDLE, 30)
```

Comment:
Do we need to set this explicitly here, or could be remove it and let it use the OS default?

Reply:
I think we could use a bit of both.

In order for Locust to function properly, you need to enable TCP Keep Alive even if it is disabled as an OS default setting.

However, the Keep Alive Idle time is a different story. In my case, my environment firewall Idle Time value was shorter than OS default, hence I needed to make it explicitly shorter.

Alternatively, we could add another optional command line parameter to set TCP_KEEPALIVE_IDLE when needed.

Reply:
Ok! I don't see how it could have any impact on the performance so it's probably fine to set it to 30. 

Unless there are cases where turning on keep-alives could have any negative effect, I'd prefer to not expose an additional command line option.